kshebeko/lesson4/src/com/company/kshebeko_class/ForestFlower.java

```diff
@@ -0,0 +1,47 @@
+package com.company.kshebeko_class;
+
+public class ForestFlower extends Flower {
+    public int stemLength = 10;
+    public String flowerBud;
+
+
+    public ForestFlower(int stemLength) {
+        this.stemLength = stemLength;
+    }
+
+    public ForestFlower(String flowerBud) {
+        this.flowerBud = flowerBud;
+    }
+
+    public ForestFlower(String name, int price){
+        super(name, price);
+        System.out.println("Forest flower : " + this.getName() + this.setName());
```

Comment:
printing should be done in main OR in special methods, not in constructor - SRP
if you need to print info about forest flower - use toString or create smth like printInfo()

Reply:
still not fixed